cmd/util/ledger/migrations/cadence_value_validation_test.go

```diff
@@ -57,6 +58,7 @@ func TestValidateCadenceValues(t *testing.T) {
 				address,
 				[]*ledger.Payload{accountStatusPayload},
 				util.RuntimeInterfaceConfig{},
+				snapshot.IndexMapBased,
```

Comment:
nit: maybe use `snapshot.MapBased` for cadence value diff since it is read-only snapshot.

Same comment for other snapshot types in this file.

```suggestion
				snapshot.MapBased,
```